bolt/test/X86/Inputs/dwarf5-df-input-lowpc-ranges-other.s

```diff
@@ -0,0 +1,453 @@
+# clang++ -fbasic-block-sections=all -ffunction-sections -g2 -gdwarf-5 -gsplit-dwarf
+# __attribute__((always_inline))
+# int doStuffOther(int val) {
+#   if (val)
+#     ++val;
+#   return val;
+# }
+# __attribute__((always_inline))
+# int doStuffOther2(int val) {
+#   int foo = 3;
+#   return val + foo;
+# }
+#
+#
+# int mainOther(int argc, const char** argv) {
```

Comment:
Are you sure mainOther was part of your cpp?

Reply:
Yes, this one wasn't working until I regenerated it.

Reply:
I don't see mainOther symbol in your assembly....